packages/did-auth-siop-op-authenticator/package.json

```diff
@@ -6,17 +6,17 @@
   "types": "dist/index.d.ts",
   "veramo": {
     "pluginInterfaces": {
-      "IDidAuthSiopOpAuthenticator": "./src/types/IDidAuthSiopOpAuthenticator.ts"
+      "IDidAuthSiopOpAuthorizer": "./src/types/IDidAuthSiopOpAuthorizer.ts"
     }
   },
   "scripts": {
     "build": "tsc --build"
   },
   "dependencies": {
-    "@sphereon/did-auth-siop": "0.2.14",
+    "@sphereon/did-auth-siop": "0.3.0-unstable.1",
     "@sphereon/pex": "1.1.4",
     "@sphereon/ssi-sdk-core": "^0.8.0",
-    "@sphereon/ssi-types": "0.8.1-next.6",
+    "@sphereon/ssi-types": "0.8.0",
```

Comment:
^0.8.0 instead of a fixed ssi-types. The previous one had a fixation because of all the backwards incompatible changes we made. This version should use the latest packages again

Reply:
fixed